shared/README.md

```diff
@@ -0,0 +1,110 @@
+# Mobility Data Specification: **Shared Definitions**
+
+This specification contains a collection of definitions, data types, and vehicle state machine information used to specify the digital relationship between *mobility as a service* Providers and the Agencies that regulate them.  These definitions are used by the other MDS APIs, including the Provider API, the Agency API, and the Policy API.
+
+* Date 11 May 2020
+* Version: ALPHA
+
+## Table of Contents
+
+* [Timestamps](#timestamps)
+* [Strings](#strings)
+* [UUIDs](#uuids)
+* [Vehicle Types](#vehicle-types)
+* [Propulsion Types](#propulsion-types)
+* [Vehicle States](#vehicle-states)
+* [Vehicle Events](#vehicle-events)
+* [Costs and Currencies](#costs-and-currencies)
+
```

Comment:
I would love to see *Geographic Data* added and reconciled between what is in `provider` and `geographies` definitions.  I don't think there is much difference.  Both are GeoJSON based.  Maybe it is just an alignment and not a shared geo standard, but it seems to fit here.

Reply:
I would put that in the Geography spec perhaps?